app/src/main/java/tttTDD/Java/Game.java

```diff
@@ -89,9 +85,9 @@ private boolean isThereAWinner() {
         return board.hasWinningSet(currentPlayer.getMarker());
     }
 
-    private void initializePlayers(boolean areWePlayingComputer) {
-        player1 = new Human("X");
-        player2 = areWePlayingComputer ? new Computer("O") : new Human("O");
+    private void initializePlayers(GameConfiguration gameConfig) {
+        player1 = gameConfig.getPlayer1();
+        player2 = gameConfig.getPlayer2();
```

Comment:
I don't love how the redundancy here, where Game holds direct reference to each Player, and, separately, a reference to the GameConfiguration that also holds a reference to each Player.